python_modules/dagster/dagster/core/asset_defs/asset_group.py

```diff
@@ -475,6 +476,31 @@ def from_current_module(
             check.failed("Could not find a module for the caller")
         return AssetGroup.from_modules([module], resource_defs, executor_def)
 
+    def materialize_in_process(
+        self, selection: Optional[Union[str, List[str]]] = None
+    ) -> ExecuteInProcessResult:
+        """
+        Executes an in-process run that materializes all assets in the group.
+
+        "In-process" means that ops are invoked without creating subprocesses. The assets can still
```

Comment:
Maybe further specify the asset-op connection for people coming to Dagster asset-first, e.g.:

"In-process means that the ops that underlie assets are invoked..."

Reply:
Good idea